Documentation/project-docs/platform-names.md

```diff
@@ -4,6 +4,26 @@ Over the years, we've accumulated quite a few names, platforms, and components t
 
 This will also list some aliases. As you'll see the aliases aren't always correct -- they are merely listed to help you find the better and less confusing terminology.
 
+In this document, the following terms are used:
+
+* **IL**. Intermediate language. IL is a stack-based instruction set and is what higher level languages, such as C#, compile down to.
+
+* **GC**. Garbage collector. Garbage collection is an implementation of automatic memory management. .NET currently uses a generational garbage collector, i.e. it groups objects into generations to limit the number of nodes it has to walk for determining which objects are alive. This speeds up collection times.
```

Comment:
> Since Mono does have other types of GCs I believe

Mono has two GCs: classic Boehm and SGen. SGen has been the default since a few years now and is also a generational collector.

+1 to your suggestion, makes it clearer.